graphdatascience/session/gds_sessions.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, api_credentials: AuraAPICredentials) -> None:
         Args:
             api_credentials (AuraAPICredentials): The Aura API credentials used for establishing a connection.
         """
-        aura_api = AuraApi(api_credentials.client_id, api_credentials.client_secret, api_credentials.tenant_id)
+        aura_api = AuraApi.create(api_credentials.client_id, api_credentials.client_secret, api_credentials.tenant_id)
```

Comment:
💡 could keep using the constructor and instead of `.create` method provide a helper that determines URI from environment:
```python
aura_api = AuraApi(base_uri=base_uri(env="..."), ...)
```

Reply:
or 
```python
base_uri = determine_base_uri(env="...")
aura_api = AuraApi(base_uri=base_uri, ...)
```

Reply:
would also work, but I am slightly leaning towards the static method. Nearly always you want to have the same logic.

Reply:
Sure, in this case I'd probably use `classmethod` instead. Name `new` would also be shorter ;) 
```
    @classmethod
    def create(cls: Type[AuraApi], client_id: str, client_secret: str, tenant_id: Optional[str] = None) -> AuraApi:
        ...
        return cls(base_uri, client_id, client_secret, tenant_id)
```